### PR TITLE
Add a repeater flow for parental leave periods.

### PR DESCRIPTION
**Pull request by arnifreyrs**
## What
Rework repeaters, screens are constructed from how many answers have already been answered as part of this repeater.

Fix webpack for application-system-api.

Be able to delete repeater items.

## Why

Needed for parental leave

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by dabbeg — APPROVED** [review]
LGTM

File: libs/application/templates/parental-leave/src/lib/ParentalLeaveForm.ts [hunk 1]
```diff
@@ -388,17 +388,30 @@ export const ParentalLeaveForm: Form = buildForm({
...
+                'You can choose to end the parental leave no later than 18 months after the date of birth.',
+              children: [
+                buildDateField({
+                  id: 'endDate',
+                  name: 'End date',
+                  placeholder: 'Pick the end date',
+                }),
+              ],
+            }),
+            // buildCustomField(
```
Comment (madebynoam) [inline libs/application/templates/parental-leave/src/lib/ParentalLeaveForm.ts:404]: Should we remove commented out part?

**Review by madebynoam — APPROVED** [review]
Nice, lgtm!

File: libs/application/templates/parental-leave/src/lib/ParentalLeaveForm.ts (hunk 1 above)
Comment (arnifreyrs) [reply libs/application/templates/parental-leave/src/lib/ParentalLeaveForm.ts:404]: This is subject to change when I combine start and end date pickers into one screen. I will trim it then

**Pull request merged**